<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Publications | Rakesh Kumar, PhD</title>
  <link rel="stylesheet" href="assets/style.css">
</head>

<body>
  <header>
    <h1>Publications</h1>
    
<div class="social-links">
  <a href="https://orcid.org/0000-0003-0811-5657" target="_blank">ORCID</a>
  <a href="https://scholar.google.com/citations?user=K48eTSEAAAAJ&hl=en" target="_blank">Google Scholar</a>
  <a href="https://www.linkedin.com/in/rakeshkpb87" target="_blank">LinkedIn</a>
</div>
    
    <nav>
      <a href="index.html">Home</a>
      <a href="about.html">About</a>
      <a href="research.html">Research</a>
      <a href="publications.html">Publications</a>
      <a href="cv.html">CV</a>
    </nav>
  </header>

  <section class="content">
    <h2>Selected Publications</h2>

    <ul>
      <li><strong>The Annotated Blueprint: Integrated functional genomic resources for a model tetraploid wheat Triticum turgidum cv. Kronos. PREPRINT (Version 1) available at bioRxiv. https://doi.org/10.1101/2025.09.12.675711 </strong> (2025)</li>

      <li>Engineering the plant intracellular immune receptor Sr50 to restore recognition 2 of the AvrSr50 escape mutant. PREPRINT (Version 1) available at bioRxiv. https://doi.org/10.1101/2024.08.07.607039 (2025)</li>

      <li>Comprehensive approaches to design efficient gRNA for SDN1-CRISPR/Cas9 genome editing in wheat. Front Genome Ed. https://doi.org/10.3389/fgeed.2025.1579165 (2025)</li>

      <li>Durum wheat mutants with enhanced disease resistance to stripe rust show differential responses to other fungal diseases. Mol Breeding.  https://doi.org/10.1007/s11032-025-01576-y  (2025)</li>

      <li>Pigments to precision: RUBY aiding genetic transformation and genome editing in wheat and barley. Physiol Mol Biol Plants. https://doi.org/10.1007/s12298-025-01591-5 (2025)</li>

      <li>A single NLR gene confers resistance to leaf and stripe rust in wheat. Nat Commun. https://doi.org/10.1038/s41467-024-54068-6 (2024)</li>

      <li>Chromosome-level genome assembly of Triticum turgidum var 'Kronos' [Data set]. Zenodo. https://doi.org/10.5281/zenodo.10215402 (2023)</li>

      <li>Evidences and opportunities for developing non-transgenic genome edited crops using the site-directed nuclease-1 approach. Crit Rev Biotechnol https://doi.org/10.1080/07388551.2023.2270581 (2023)</li>

      <li>Gas exchange tracks root hydraulic conductivity which in term regulated by plasma membrane AQPs in the model plant Setaria Viridis. Plant Physiol https://doi.org/10.1093/plphys/kiad469 (2023)</li>

      <li>Explicating drought tolerance of wheat (Triticum aestivum L.) through stress tolerance matrix. Plant Physiol Rep https://doi.org/10.1007/s40502-022-00707-3 (2023)</li>

      <li>Physio-biochemical characterization of wheat genotypes under temperature stress. Physiol Mol Biol Plants. https://doi.org/10.1007/s12298-022-01267-4 (2022)</li>

      <li>CRISPR/Cas technology for improving nutritional values in the agricultural sector: an update. Mol Biol Rep 49: 7101-7110 https://doi.org/10.1007/s11033-022-07523-w (2022)</li>

      <li>Identification and characterization of multiple abiotic stress tolerance genes in wheat. Mol Biol Rep 47: 8629-8643. https://doi.org/10.1007/s11033-020-05906-5 (2020)</li>

      <li>CRISPR-based genome editing in wheat: A comprehensive review and future prospects. Mol Biol Rep 46(3): 3557–3569. https://doi.org/10.1007/s11033-019-04761-3 (2019)</li>

      <li>Optimization of Agrobacterium-mediated transformation in spring bread wheat using mature and immature embryos. Mol Biol Rep 46(2): 1845-1853. https://doi.org/10.1007/s11033-019-04637-6 (2019)</li>

      <li>Development of an efficient and reproducible regeneration system in wheat (Triticum aestivum L.). Physiol Mol Biol Plants 23(4): 945-954. https://doi.org/10.1007/s12298-017-0463-6 (2017)</li>

      <li>Synergistic effect of cefotaxime and timentin to suppress the Agrobacterium overgrowth in wheat (Triticum aestivum L.) transformation. Asian J Microbiol Biotechnol Environ Sci 19(4): 961-967 (2017)</li>
      
      <li>Genetic diversity analysis of Lepidium sativum (Chandrasur) using inter simple sequence repeat (ISSR) markers. J For Res 26(1): 107–114. https://doi:10.1007/s11676-014-0545-z (2015)</li>
      
      <li>Genetic transformation of wheat-Present status and future potential. J Wheat Res 6(2): 1-13. (2014)</li>
     
      <li>An assessment of genetic fidelity in Lepidium sativum regenerants using RAPD markers. Eco Env Cons 20(1): 341-347. (2014)</li>
      
    </ul>
  </section>

  <footer>
    © 2025 Rakesh Kumar, PhD
  </footer>
</body>
</html>
